GoToProject/android/app/src/main/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+  <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">AIzaSyB4SwDlMm4Dr2c7j5A-iRbFbAWxIKO-tOA</string>
```

Comment:
Is there a way you can hide your API Key from this section?

Reply:
API key has been removed from visible files, I created and env file out of git